### PR TITLE
Basic medical module options

### PR DESCRIPTION
**Pull request by KoffeinFlummi**

### CODE REVIEW

**Comment by ViperMaul** [comment]
Looks like this PR needs some more love.

**Comment by KoffeinFlummi** [comment]
:heart:

**Comment by KoffeinFlummi** [comment]
Should be ready. @Glowbal

File: addons/medical/functions/fnc_getBloodLoss.sqf [hunk 1]
```diff
@@ -17,13 +17,15 @@
 
 private ["_totalBloodLoss","_tourniquets","_openWounds", "_cardiacOutput", "_internalWounds"];
 // TODO Only use this calculation if medium or higher, otherwise use vanilla calculations (for basic medical).
+
+_unit = _this select 0;
```
Comment (Glowbal) [inline addons/medical/functions/fnc_getBloodLoss.sqf]: Missing private

**Comment by Glowbal** [comment]
:+1:

**Pull request merged**